src/main/java/com/mate/fruitshop/service/impl/ReportCreatorServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package com.mate.fruitshop.service.impl;
+
+import com.mate.fruitshop.dao.FruitStorageDao;
+import com.mate.fruitshop.dao.impl.FruitStorageDaoImpl;
+import com.mate.fruitshop.model.FruitEntry;
+import com.mate.fruitshop.service.ReportCreatorService;
+import java.util.List;
+
+public class ReportCreatorServiceImpl implements ReportCreatorService {
+
+    public static final String REPORT_HEADER = "fruit,quantity";
+    public static final String DELIMITER = ",";
+
+    @Override
+    public String createReport() {
+        FruitStorageDao dao = new FruitStorageDaoImpl();
+        List<FruitEntry> entries = dao.getAllFruitEntries();

```

Comment:
Let's put the list of enties as method parameter `createReport(List<FruitEntry> fruits)`. This make the method more agile and it can be reusable in future.